Sources/Testing/Running/Configuration.swift

```diff
@@ -105,14 +105,13 @@ public struct Configuration: Sendable {
   /// By default, the value of this property allows for a single iteration.
   public var repetitionPolicy: RepetitionPolicy = .once
 
-  // MARK: - Main actor isolation
+  // MARK: - Isolation context for synchronous tests
 
-#if !SWT_NO_GLOBAL_ACTORS
-  /// Whether or not synchronous test functions need to run on the main actor.
+  /// The isolation context to use for synchronous test functions.
   ///
-  /// This property is available on platforms where UI testing is implemented.
-  public var isMainActorIsolationEnforced = false
-#endif
+  /// If the value of this property is `nil`, synchronous test functions run in
+  /// an unspecified isolation context.
+  public var defaultIsolationContext: (any Actor)? = nil
```

Comment:
I'd find it helpful if the name of this property indicated that it only applies to _synchronous_ functions, not async ones. (Because asynchronous test functions could, hypothetically, honor this too, right? Not that we want them to.) Maybe drop the word "context" to make space for the word "synchronous"?
```suggestion
  public var defaultSynchronousIsolation: (any Actor)? = nil
```

Reply:
The naming for this concept is a bit vague on the Swift side, I think, but in general it's an "isolation context", so we could write `defaultSynchronousIsolationContext`?